### DISCUSSION TITLE
Is it possible to run scp in QGIS python console?

### DISCUSSION THREAD

**Discussion by talentstuder** [semiautomaticgit/SemiAutomaticClassificationPlugin#191] [General]
When we run a function in Toolbox, there will be a history generated in the Processing-History. The history can show the python code so that we can use it on other environment. However, the scp plugin is not in the Toolbox and there is no history generated when we finish a classification. My question is: is it possible to get the python code of scp plugin? So that I can use the code for further use.

**Comment by semiautomaticgit** [comment]
Hello @talentstuder , unfortunately at the moment SCP functions are not available through Python directly.
You can use Batch functions to automate the processing https://semiautomaticclassificationmanual.readthedocs.io/en/latest/batchTab.html
